credoai/evaluators/utils/fairlearn.py

```diff
@@ -16,47 +17,80 @@ def create_metric_frame(metrics, y_pred, y_true, sensitive_features):
     )
 
 
+def filter_processed_metrics(
+    processed_metrics, metric_categories=None, Xmetric_categories=None, takes_prob=None
+):
+    """Filters processed metrics
```

Comment:
As part of the standardization of the docstrings, I had chosen to put the title in a new line:

```
"""
Filters processed metrics
```
This can be a setting in VSCode formatter, so we don't have to remember doing it.

It is just a stylistic choice in general, the only place where it matters is in evaluators docstrings, since they are parsed to create documentation on RTD.